apps/frontend/src/pages/Homepage.tsx

```diff
@@ -113,6 +115,15 @@ const Homepage = () => {
             console.error("Error fetching listings:", error);
           });
         setSearchPerformed(false);
+        _axios_instance
+          .get("/charities/current")
```

Comment:
async/await

Reply:
I kept it the same to stay consistent with the rest of the application